communities.md

```diff
@@ -100,3 +100,83 @@ Example: not available
 The supported parameters are:
 * **(mandatory)** parent, the UUID of the parent community
 * page, size [see pagination](README.md#Pagination)
+
+## Creating communities
+
+### Creating top level community
+
+**POST /api/core/communities**
+
+To create a top level community, perform as post with the JSON below to the communities endpoint when logged in as admin.
+
+```
+{
+    "name": "test creation",
+    "metadata": [
+        {
+            "key": "dc.title",
+            "value": "test creation",
+            "language": null
+        }
+    ]
+}
+```
+
+### Creating subCommunity
+
+**POST /api/core/communities?parentCommunity=<:communityUUID>**
+
+To create a sub level community, perform as post with the JSON below to the communities endpoint when logged in as admin.
+
+```
+{
+    "name": "test subcommunity",
+    "metadata": [
+        {
+            "key": "dc.title",
+            "value": "test subcommunity",
+            "language": null
+        }
+    ]
+}
+```
+
```

Comment:
please specify what happen if an not existent parent community is used. I suggest 422 UNPROCESSABLE ENTITY as 404 should be used to say that the endpoint doesn't exist

Reply:
The error messages with an explanation have been included for PUT and POST for collections and communities. I'd recommend to also add the 422 UNPROCESSABLE ENTITY to 
https://github.com/DSpace/Rest7Contract#on-the-naming-of-endpoints (similar to your recommendation on parameters below)